bokehjs/src/coffee/common/plot.coffee

```diff
@@ -122,17 +122,17 @@ class PlotView extends ContinuumView
   
   init_webgl: () ->
 
-    # Get visible and invisible canvas 
-    # (doing it like this makes it easy to swap them during debugging)    
-    glcanvas = document.createElement('canvas')
-    
-    # Create GL context to draw to
+    # We use a global invisible canvas and gl context. By having a global context,
+    # we avoid the limitation of max 16 contexts that most browsers have. 
+    glcanvas = window._bokeh_gl_canvas
```

Comment:
Oh sure, as long as its "global" to all plots in a browser window. How would I use a module-level variable? Is there a global keyword in coffeescript?